etcd.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: etcd
+  version: 3.5.6
+  epoch: 0
+  description: A highly-available key value store for shared configuration and service discovery.
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - git
+      - bash
+      - go
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/coreos/etcd/archive/v${{package.version}}.tar.gz
```

Comment:
nitpick: shouldn't it be https://github.com/etcd-io/etcd? coreos/etcd redirect to the new one anyway

Reply:
Fixed!